build.gradle.kts

```diff
@@ -26,5 +26,11 @@ allprojects {
 
     tasks.withType<org.jetbrains.kotlin.gradle.dsl.KotlinCompile<*>> {
         kotlinOptions.allWarningsAsErrors = true
+        kotlinOptions.freeCompilerArgs += "-Xexpect-actual-classes"
+        kotlinOptions.freeCompilerArgs += "-Xwasm-enable-array-range-checks"
```

Comment:
Better to provide this option only to tasks that support it

Reply:
I put this argument for KotlinJsCompile task. There seems for now there is no a good way to put it into Wasm only compilation.